spec/stat_tracker_spec.rb

```diff
@@ -120,16 +151,4 @@
     end
   end
 
-	  it 'can count the total number of teams' do
```

Comment:
I moved these three tests up into the game statistics describe block so they are with the other tests related to them